scripts/get-filename.js

```diff
@@ -5,15 +5,19 @@
  */
 
 import process from 'node:process';
-import { titleToSlug } from '../sdk.mjs';
+import {titleToSlug} from '../sdk.mjs';
 
 if (process.argv.length < 3) {
   console.error('Provide a brand name as argument');
   process.exit(1);
 } else {
   const brandName = process.argv
     .slice(3)
-    .reduce((acc, arg) => `${acc} ${arg}`, process.argv[2]);
+    // eslint-disable-next-line unicorn/no-array-reduce
+    .reduce(
+      (accumulator, argument) => `${accumulator} ${argument}`,
+      process.argv[2],
+    );
```

Comment:
This code is a complete madness, please change it by just:

```js
const brandName = process.argv[2];
```

Reply:
Haha, sure.